### ISSUE TITLE
paste-node only works "locally"

### ISSUE THREAD

**Issue by edreamleo** [boltex/leointeg#151]
On Windows, paste-node works, but only after copy-node *within leoInteg*. That is, copy-node in Leo, followed by paste-node in leoInteg apparently does nothing.

However, copy-node in Leo, followed by paste-*text* in leoInteg *does* work. Something like the following appears in leoInteg's body pane:
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Created by Leo: http://leoeditor.com/leo_toc.html -->
<leo_file xmlns:leo="http://leoeditor.com/namespaces/leo-python-editor/1.1" >
<leo_header file_format="2"/>
<vnodes>
<v t="ekr.20210611084045.71"><vh>server._makePackage</vh></v>
</vnodes>
<tnodes>
<t tx="ekr.20210611084045.71">def _makePackage(self, package=None):
    if package is None:
        package = {}
    package["id"] = self.currentActionId
    if 1:
        print(
            f"{self.currentActionId:&gt;3} "
            f"{self.action or 'No Action':&gt;30} "
            f"{','.join(list(package.keys()))}", flush=True)
    return(json.dumps(package, separators=(',', ':')))  # send as json
</t>
</tnodes>
</leo_file>
```

In short, leoInteg does not appear to be using the system clipboard for paste-node.

**Comment by boltex** [comment]
Duplicate of #21 - seems like you had commented on it too with some pointer to a solution for this. I just didn't have time to fix this one yet :)

**Comment by boltex** [comment]
Closing as it is a duplicate.
